### PR TITLE
Write voucher documentation

### PR DESCRIPTION
**Pull request by chrissvo**
Fixes: https://github.com/mgmco/bugs/issues/1859

### CODE REVIEW

File: _sales_tracking/vouchers.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+</a>
+
+
+Implementation requires the client to:
+
+* Have the ability to redeem vouchers on the website of the client
+* Provide MGMco with a set of buyer vouchers
+* Provide MGMco with a set of promoter vouchers
+
+From the technical perspective there is no need to store an `MGMco sale ID` on the client side. The promoter is identified by the buyer voucher.
```
Comment (harianus) [inline _sales_tracking/vouchers.md]: So they have to save the `voucher_id` instead?

Comment (chrissvo) [inline _sales_tracking/vouchers.md]: No, they have to save nothing. Just return the voucher used + email buyer. The voucher used is tied to a promoter.

Comment (harianus) [inline _sales_tracking/vouchers.md]: But when the email changes on the way? Would we lose them?

Comment (chrissvo) [inline _sales_tracking/vouchers.md]: Nope, the buyer does not exist until he gives his emailaddress to our client.

Comment (harianus) [inline _sales_tracking/vouchers.md]: Owja, I get it.

**Pull request merged**